.github/workflows/npm-publish.yml

```diff
@@ -28,7 +28,7 @@ jobs:
       - uses: actions/setup-node@v3
         with:
           node-version: 16
-          registry-url: https://registry.npmjs.org/
+          registry-url: https://registry.npmjs.com/
```

Comment:
why doesn't .org work?

Reply:
They seem to be interchangeable, we have just have to match whatever we put in in the `publishConfig.registry` field in our `package.json` with above field. Because we initially it as "https://registry.npmjs.com/" in `package.json`, I was syncing the workflow file here to match that. 

However, we can switch both back to .org and I believe it should work just fine as well. I will push an update

Reply:
thanks, i just wanted to understand why .org didn't work.

Reply:
Scratch that, they might not be interchangeable. That's really strange, in my forked version of this repo, I was able to change them with no problem

Reply:
Nvm, they are interchangeable (.com vs. .org). I finally resolved the bug, and I had sent this assuming the .com / .org change was causing the final failure, but it was from TechOps's side. The secret was getting wrapped in a JSON object on AWS for some reason instead of being stored as plain text.